lib/slax_web/websockets/poker.ex

```diff
@@ -89,7 +89,7 @@ defmodule SlaxWeb.Poker do
         estimates =
           round.estimates
           |> Enum.sort_by(& &1.value)
-          |> Enum.map(&"#{&1.user} (#{&1.value})#{if &1.reason, do: ': #{&1.reason}'}")
+          |> Enum.map(&"#{&1.user} (#{&1.value})#{if &1.reason, do: ~c": #{&1.reason}"}")
```

Comment:
Anybody know why we're using charlists here?

Reply:
That's really funky

Reply:
It also shows up as funky from the syntax highlight in my vscode now; but it does still work

Reply:
wait that's just being the `" ` and the `:` are so close together